internal/migrate/stage_contract.go

```diff
@@ -66,94 +73,251 @@ func stageContract(
 		return nil, err
 	}
 
-	contractName := args[0]
-	contract, err := state.Contracts().ByName(contractName)
+	// Validate command arguments
+	optionCount := boolCount(stageContractflags.All, len(stageContractflags.Accounts) > 0, len(args) > 0)
+	if optionCount > 1 {
```

Comment:
We may also want a max limit. Some arbitrary high enough number not to overwhelm things.